### DISCUSSION TITLE
Vinegar Studio Version History

### DISCUSSION THREAD

**Discussion by mrfiguy** [vinegarhq/vinegar#437] [Q&A] [answered]
Does anybody have a list of older versions of roblox studio for vinegar? I've been trying to look for older versions to see if they're compatible.

**Comment by mrfiguy** [comment]
I'm specifically looking for versions dating back to june of 2024, but any rage is acceptable.

**Comment by sewnie** [comment]
https://github.com/vinegarhq/vinegar/issues/435#issuecomment-2374365663

**Reply by mrfiguy** [reply]
this is not for the recent error showing up on the latest version btw

**Reply by sewnie** [reply, marked as answer]
https://setup.rbxcdn.com/DeployHistory.txt
